apis/r/src/Makevars.in

```diff
@@ -1,7 +1,9 @@
 CXX_STD = CXX17
 
-## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS
-PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ @cxx17_macos@ -Iinclude
+## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS.
+## Our build assumes C++ libtiledbsoma in /usr/local/include and /usr/local/lib. Here we explicitly
+## add -I and -L for those, since while some compilers add these paths implicitly, others do not.
+PKG_CPPFLAGS = -Wno-deprecated-declarations -I. -I../inst/include/ -I/usr/local/include -L/usr/local/lib @tiledb_include@ @cxx17_macos@ -Iinclude
```

Comment:
`PKG_CPPFLAGS` is for the preprocessor.  The linker options do not belong, they go to `PKG_LIBS`.